code/modules/clothing/gloves/_gloves.dm

```diff
@@ -11,6 +11,7 @@
 	var/transfer_prints = FALSE
 	strip_delay = 20
 	equip_delay_other = 40
+	equip_delay_self = 0.6 SECONDS
```

Comment:
oh no, a 0.6 second delay. this will definitely stop me from combat-swapping my gloves